src/datajudge/db_access.py

```diff
@@ -648,7 +649,19 @@ def get_column(
 
     if not aggregate_operator:
         selection = sa.select([column])
-        result = engine.connect().execute(selection).scalars().all()
+
+        if is_snowflake(engine):  # check if we have a snowflake cursor
+            snowflake_cursor: SnowflakeCursor = engine.connect().connection.cursor()
+
+            # note: this step requires pandas to be installed
+            pa_table = snowflake_cursor.execute(str(selection)).fetch_arrow_all()
+            if pa_table:  # snowflake connector returns NoneType when the table is empty
+                result = pa_table.column(0).to_numpy()
```

Comment:
> Just accessing the first column looks unsafe, but should ideally be always correct, since our `selection` only selects a single column.

That's fine.